index.html

```diff
@@ -200,6 +200,27 @@ <h2 class="big-text">About Myself</h2>
           <hr>
         </div>
     </section>
+    <section class="contact-section">
+      <div class="contact-Me">
+        <h3 class="contact">Contact Me</h3>
+        <p class="last-text">If you have an application you are interested in developing, a feature that you need built or a project that
+          needs coding. I’d love to help with it</p>
+        <form action="https://formspree.io/f/xvolqjkn" method="post" id="contact-form">
+          <label for="name" id="name-label"></label>
+          <input  id="name" type="text" required placeholder="Enter your Name">
```

Comment:
- [ ] As stated in the project requirements: Please ensure that the name input accepts up to 30 chars. You can do this by adding the `maxlength="30"` attribute.